doc/source/user/quickstart.rst

```diff
@@ -185,27 +185,27 @@ state of the memory. By default, the dtype of the created array is
     array([[0., 0., 0., 0.],
            [0., 0., 0., 0.],
            [0., 0., 0., 0.]])
-    >>> np.ones( (2,3,4), dtype=np.int16 )                # dtype can also be specified
+    >>> np.ones((2, 3, 4), dtype=np.int16)  # dtype can also be specified
     array([[[1, 1, 1, 1],
             [1, 1, 1, 1],
             [1, 1, 1, 1]],
     <BLANKLINE>
            [[1, 1, 1, 1],
             [1, 1, 1, 1],
             [1, 1, 1, 1]]], dtype=int16)
-    >>> np.empty( (2,3) )                                 # uninitialized
-    array([[  3.73603959e-262,   6.02658058e-154,   6.55490914e-260],  # may vary
-           [  5.30498948e-313,   3.14673309e-307,   1.00000000e+000]])
+    >>> np.empty((2, 3))  # uninitialized, the result may vary
+    array([[3.73603959e-262, 6.02658058e-154, 6.55490914e-260],
+           [5.30498948e-313, 3.14673309e-307, 1.00000000e+000]])
```

Comment:
"may vary" is a special marker understood by our CI tools, I think you might need to put it back where it was.

Reply:
Interesting that this did not fail.

Reply:
This is failing in other PRs now, eg https://app.circleci.com/pipelines/github/numpy/numpy/5410/workflows/93d901be-60df-4a79-9d56-a35bdcc5b2f2/jobs/17498/parallel-runs/0/steps/0-106

Reply:
potential fix in gh-18008